docs/conf.py

```diff
@@ -8,58 +8,60 @@
 
 # Insert module path here
 sys.path.insert(0, abspath(dirname(__file__)))
-sys.path.insert(0, abspath(join(dirname(__file__), '..', 'lib')))
+sys.path.insert(0, abspath(join(dirname(__file__), "..", "lib")))
 
 from unittest import mock
-sys.modules['wpilib'] = mock.MagicMock()
-sys.modules['hal_impl'] = mock.MagicMock()
-sys.modules['hal_impl.data'] = mock.MagicMock()
-sys.modules['hal_impl.sim_hooks'] = mock.MagicMock()
-sys.modules['hal_impl.types'] = mock.MagicMock()
-sys.modules['hal_impl.mode_helpers'] = mock.MagicMock()
 
-sys.modules['tkinter'] = mock.MagicMock()
+sys.modules["wpilib"] = mock.MagicMock()
+sys.modules["hal_impl"] = mock.MagicMock()
+sys.modules["hal_impl.data"] = mock.MagicMock()
+sys.modules["hal_impl.sim_hooks"] = mock.MagicMock()
+sys.modules["hal_impl.types"] = mock.MagicMock()
+sys.modules["hal_impl.mode_helpers"] = mock.MagicMock()
+
+sys.modules["tkinter"] = mock.MagicMock()
 
 import pyfrc.config
-pyfrc.config.config_obj['pyfrc'] = dict(game_specific_messages=[])
+
+pyfrc.config.config_obj["pyfrc"] = dict(game_specific_messages=[])
 
 # -- RTD configuration ------------------------------------------------
 
 # on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
-on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+on_rtd = os.environ.get("READTHEDOCS", None) == "True"
```

Comment:
I thought you were going to leave this alone. Are we using double quotes now?

Reply:
So, I thought about it. I dislike double quotes -- but I also recognize that it's much easier to tell a user `black .` instead of `black -S .` ... and if they accidentally do the latter, then it's possible that they might not have the git-fu to fix their problem.

This could be alleviated by providing a makefile or a script for users to run, but that seems annoying too.

I've been doing golang a lot lately, and it's been really nice to have an autoformatter fix things for me automatically, so I'm looking forward to that experience with python too.

Reply:
Does black get rid of whitespace on blank lines 👀 ?

Reply:
:shipit: